structured/structured.proto

```diff
@@ -84,3 +84,16 @@ message TableDescriptor {
   optional uint32 next_index_id = 6 [(gogoproto.nullable) = false,
       (gogoproto.customname) = "NextIndexID"];
 }
+
+// DatabaseDescriptor represents a namespace (aka database) and is stored
+// in a structured metadata key. The DatabaseDescriptor has a globally-unique
+// ID shared with the TableDescriptor ID.
+// Permissions are applied to all tables in the namespace.
```

Comment:
nope. For now tables do not have permissions. The next step is to apply permissions at the database level. After that comes per-table permissions that are inherited from the database-level permissions.